### PR TITLE
Fix: sweep step export

### PR DESCRIPTION
**Pull request by WirajanDASILVA**
* **Please check if the PR fulfills these requirements**
- [X] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [X] Bug fix:
- [ ] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:

### CODE REVIEW

File: volmdlr/faces.py [hunk 1]
```diff
@@ -6975,7 +6975,11 @@ def to_step(self, current_id):
         step_content = ''
         face_ids = []
         for face in self.faces:
-            face_content, face_sub_ids = face.to_step(current_id)
+            if isinstance(face, Face3D):
+                face_content, face_sub_ids = face.to_step(current_id)
+            else:
```
Comment (sbendjebla) [inline volmdlr/faces.py:6980]: Why it is different when the face is not a Face3D ?

File: volmdlr/wires.py [hunk 2]
```diff
@@ -4373,17 +4373,17 @@ def from_step(cls, arguments, object_dict):
         return cls.from_center_normal(center, normal, radius,
                                       arguments[0][1:-1])
 
-    def to_step(self, current_id, surface_id=None):
+    def to_step(self, current_id, **kwargs):
```
Comment (sbendjebla) [inline volmdlr/wires.py:4376]: Can you specify in the docstrings what are the `kwargs`, please ?

File: volmdlr/edges.py [hunk 3]
```diff
@@ -4376,7 +4376,7 @@ def revolution(self, axis_point: volmdlr.Point3D, axis: volmdlr.Vector3D,
             return [surface.rectangular_cut(0, angle,
                                             arc2d.angle1, arc2d.angle2)]
 
-    def to_step(self, current_id):
+    def to_step(self, current_id, **kwargs):
```
Comment (sbendjebla) [inline volmdlr/edges.py:4379]: There are no `kwargs`, I think !

File: volmdlr/faces.py (hunk 1 above)
Comment (WirajanDASILVA) [reply volmdlr/faces.py]: Face3D returns a list of ids, the ClosedShell3D returned only one id. I have changed all to_step methods to return a list of ids. I've deleted this verification

File: volmdlr/wires.py (hunk 2 above)
Comment (WirajanDASILVA) [reply volmdlr/wires.py:4381]: Done

File: volmdlr/edges.py (hunk 3 above)
Comment (WirajanDASILVA) [reply volmdlr/edges.py:4396]: I have added **kwargs to all to_step methods, so we follow a standard. What do you think?

File: volmdlr/faces.py (hunk 1 above)
Comment (WirajanDASILVA) [reply volmdlr/faces.py:6989]: When it's a face, face_sub_ids is a liste of integers. But then it is a closedShell3D, it is a single integer. We have this weird behaviour where a closedshell3d is passed as Face in faces...

**Review by sbendjebla — DISMISSED** [review]

**Review by sbendjebla — APPROVED** [review]

**Pull request merged**